.github/workflows/run.yml

```diff
@@ -14,11 +15,27 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-      - run: choco install -y au
+      - name: setup
```

Comment:
Could we update the name of the action to be more meaningful? I believe this is for windows but more context would be great.